config/manifests/osl/bases/logic-operator-rhel8.clusterserviceversion.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
     operators.openshift.io/valid-subscription: '["OpenShift Container Platform", "OpenShift
```

Comment:
shouldn't this file be removed from upstream?

@rgdoliveira  ^

Reply:
this is a long discussion and eventually they will be removed at some point and be part just of a midstream. (cc @mareknovotny

Reply:
yes, we started pointing to the specifics not related to upstream on another PR change, but for now just keep it not removed is ok